docs/projects/ansible/collection/centurion/playbooks/index.md

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
can you update your dev branch so that it's up to date. hopefully that fixes the jobs that are not running

``` bash
git remote add nfc https://github.com/nofusscomputing/ansible_collection_centurion.git
git fetch --all -fpvt
git switch development
git pull nfc development --rebase
git push origin
```

Reply:
completed this request